test/passes/optimize-instructions_all-features.txt

```diff
@@ -2832,7 +2932,7 @@
    (i32.eq
     (i32.sub
      (local.get $x32)
-     (i32.const 10)
+     (i32.const -10)
     )
     (local.get $x32)
    )
```

Comment:
Btw this just partially optimization for
```wat
    (drop
      (i32.eq
        (i32.sub
          (local.get $x32)
          (i32.const 10)
        )
        (i32.sub
          (local.get $x32)
          (i32.const 20)
        )
      )
    )
```
Will be great optimize it better to `(drop (i32.const 0))`